### PR TITLE
Add instructions for setting a site wide error

### PR DESCRIPTION
**Pull request by jmhooper**
This commit contains instructions for setting a site-wide error.

### CODE REVIEW

**Comment by wslack** [comment]
Can we call it a "site wide alert" in the documentation?

**Comment by jmhooper** [comment]
I've amended the commit to include that change

**Comment by jmhooper** [comment]
Tried the commands to update the user provided service and it didn't work. We'll need to update those.

File: pages/how-federalist-works/site-wide-alert.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+Members of the Federalist dev team can update Federalist to display a banner explaining any maintenance, bugs or degradation that users should be aware of. This is done by setting values in a user provided service.
+
+## Setting A Site Wide Alert
+
+To set a site wide alert, target the Federalist production space and run the following on the command line:
+
+```shell
+cf uups federalist-site-wide-error -p '{
+  HEADING: "There's an error",
```
Comment (jmhooper) [inline pages/how-federalist-works/site-wide-alert.md]: I've figured out what is wrong with this command.

1. The `HEADING` and `BODY` need to be in quotation marks for it to be valid JSON
2. The apostrophe in `there's` is terminating the string, creating a syntax error.

**Comment by jmhooper** [comment]
I've fixed the command and tried it out in staging to verify it works :+1:

File: pages/how-federalist-works/site-wide-alert.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+To set a site wide alert, target the Federalist production space and run the following on the command line:
+
+```shell
+cf uups federalist-site-wide-error -p '{
+  "HEADING": "There is an error",
+  "BODY": "We are working on it"
+}'
+```
+
+Once the user provided service is updated, the app will need to be restaged. Restaging the app will result in downtime, so it is recommended that instead a zero-downtime-deploy is run. The easiest way to do this is often by re-building the master branch in CI.
```
Comment (jseppi) [inline pages/how-federalist-works/site-wide-alert.md]: remove `often`

File: pages/how-federalist-works/site-wide-alert.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+---
+title: Site Wide Alert
+parent: How Federalist Works
+---
+
+Members of the Federalist dev team can update Federalist to display a banner explaining any maintenance, bugs or degradation that users should be aware of. This is done by setting values in a user provided service.
```
Comment (jseppi) [inline pages/how-federalist-works/site-wide-alert.md]: add serial comma after `bugs`

**Pull request merged**